services/dsp-b/src/public/html/ad-server-ad.html

```diff
@@ -0,0 +1,28 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>DSP-A contextual ad</title>
```

Comment:
Nit: Shouldn't this be DSP-B now?

Reply:
Good catch! Yea, I was speed coding and I've missed a lot of little things 😅